Updated PublicIpAddressCombo doc re. result value

QuasarSE opened the pull request:
Updated doc to fix inverted note about options.hideNone and to be specified about "specified" vs. specified with a value of true.

azurecla commented:
Hi __@conceptdev__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <p>The agreement was validated by Microsoft and real humans are currently evaluating your PR.</p>

TTYL, AZPRBOT;

davidwrede commented:
Thanks for the updates, Craig!

QuasarSE committed:
Updated PublicIpAddressCombo doc re. result value

Updated doc to fix inverted note about options.hideNone and to be specified about "specified" vs. specified with a value of true.
